docs/registry/_index.md

```diff
@@ -29,48 +29,22 @@ menuindent: true
 The [Viam registry](https://app.viam.com/registry) is the central place where you can browse:
 
 {{< cards >}}
-{{% manualcard link="#ml-models" %}}
-
-**ML models** to deploy with machine applications like computer vision
-
-{{% /manualcard %}}
 {{% manualcard link="#modular-resources" %}}
 
 **Modular resources** that add capabilities to your machine beyond what is built into `viam-server`
 
 {{% /manualcard %}}
-{{< /cards >}}
-
-## ML models
+{{% manualcard link="#ml-models" %}}
 
-Viam provides the ability to train, upload, and deploy machine learning models within the platform.
-See [Machine Learning](/services/ml/) for more information.
+**ML models** to deploy with machine applications like computer vision
 
-The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.
-You can also [upload your own model to the registry](/services/ml/upload-model/).
+{{% /manualcard %}}
+{{% manualcard link="#ml-training-scripts" %}}
 
-You can search the available ML models from the Viam registry here:
+**Training scripts** to train machine learning models in the Viam cloud from custom Python scripts.
```

Comment:
@tahiyasalam Is this accurate or would you know of a more descriptive way to phrase it?

Reply:
Maybe `Python source distributions used to train and produce models in the Viam cloud for custom machine learning`?

Reply:
Python source distributions sounds to me like the source code for python 3.9 o or other versions...

Reply:
That's fair, we can Python scripts